elephant/test/test_current_source_density.py

```diff
@@ -153,5 +154,29 @@ def test_kcsd2d_init(self):
         self.assertEqual(len(result.times), 1)
 
 
+class GenerateLfpTestCase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.one_dimensional = np.linspace(0, 10, 2304)
+        cls.two_dimensional = np.linspace(0, 10, 2304
+                                          ).reshape(2304, 1)
+
+    def test_generate_lfp_one_dimensional_array(self):
+        """
+        Regression test for Issue #546,
+        see: https://github.com/NeuralEnsemble/elephant/issues/546
+        """
+        # this should raise an error
```

Comment:
hopefully not any more?
```suggestion
        # this should raise NOT an error
```